src/app/lib/utilities/getRemoteConfig/index.js

```diff
@@ -0,0 +1,45 @@
+import toggles from '#lib/config/toggles';
+import constructTogglesEndpoint from '#contexts/ToggleContext/utils/constructTogglesEndpoint';
+import nodeLogger from '#lib/logger.node';
+import { CONFIG_REQUEST_RECEIVED, CONFIG_FETCH_ERROR } from '#lib/logger.const';
+import getOriginContext from '#contexts/RequestContext/getOriginContext';
+
+const logger = nodeLogger(__filename);
+
+const getRemoteConfig = async (service, cache) => {
+  const environment = process.env.SIMORGH_APP_ENV || 'local';
+  const localToggles = toggles[environment];
+  if (!localToggles.enableFetchingToggles.enabled) {
+    return null;
+  }
+
+  const { origin } = getOriginContext();
+  const url = constructTogglesEndpoint(service, origin);
+
+  const cachedResponse = cache && cache.get(url);
+  if (cachedResponse) {
+    return cachedResponse;
+  }
+
+  try {
+    logger.info(CONFIG_REQUEST_RECEIVED, { url, service });
+    const response = await fetch(url, { headers: { origin } });
```

Comment:
What happens if for some reason the remote config returns an empty object?

Reply:
It would just use the defaults in that case.